data/org.freedesktop.portal.OpenURI.xml

```diff
@@ -71,6 +71,14 @@
               The ask option was introduced in version 3 of the interface.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>activation_token s</term>
+            <listitem><para>
+              A token that can be used to activate the chosen application.
+            </para><para>
+              The activation_token option was introduced in version 4 of the interface.
+            </para></listitem>
+          </varlistentry>
```

Comment:
Probably want this as part of the app chooser spec too.

Reply:
Done